app/lib/seeder.rb

```diff
@@ -73,6 +73,9 @@ def admin_user
         admin: true,
         confirmed_at: Time.current
       )
+      puts "==== Admin User====\n"
```

Comment:
Seems like a preceding `\n` is better than the trailing one. Also missing a space:

```suggestion
      puts "\n==== Admin User ===="
```

It also might be nice maybe at the top of `Seeder#seed` to also output something like `\n==== Seeding Database ====` so there is some context for these users.

Reply:
Done